src/inspection/inspectionLinks.ts

```diff
@@ -200,3 +200,31 @@ export async function checkIfLinkedInspectionsAreUpdated(inspection: Inspection)
     ([startOfWeek, inspectionId]) => linkedInspections[startOfWeek] !== inspectionId
   )
 }
+
+// Check from which other inspections this inspection is linked.
+// eg. if arg is a pre-inspection, returns all post-inspections where it is used.
+export async function getLinkedFromInspections(inspection: Inspection): Promise<Inspection[]> {
+  let linkedFromType =
+    inspection.inspectionType === InspectionType.PRE
+      ? InspectionType.POST
+      : inspection.inspectionType === InspectionType.POST
+      ? InspectionType.EQUIPMENT_DEFECT
+      : false
+
+  if (!linkedFromType) {
+    return []
+  }
+
+  let repo = getRepository(Inspection)
+
+  let linkedFromInspections = await repo.find({
```

Comment:
Nit: could it be like this?
```suggestion
  let linkedInspections = await getRepository(Inspection).find({
```

Reply:
Sure, we can inline the repo creation. But as I said previously, "from" is an important distinction here and it tries to tell you what's going on.